packages/node_modules/@webex/webex-core/src/lib/credentials/credentials.js

```diff
@@ -129,6 +130,28 @@ const Credentials = WebexPlugin.extend({
     /* eslint-enable camelcase */
   },
 
+  /**
+   * Extract the OrgId [realm] from a provided JWT.
+   *
+   * @param {string} token - The JWT to extract the OrgId from.
+   * @returns {string} - The OrgId.
+   */
+  extractOrgIdFromJWT(token = '') {
+    // Split the provided JWT into subsections.
+    const fields = token.split('.');
+
+    // Validate that the provided token is a JWT.
+    if (fields.length !== 3) {
+      return undefined;
```

Comment:
Should you return empty string `''` or `null`?

Reply:
Needs to be null in this case, later we will do a:
```js
combinedFunctionForGettingOrgId(token = '') {
  const foundOrgId = extractFromJWT(token) || extractFromUserToken(token);

  if (!foundOrgId) {
    throw new Error('message');
  }
  ...
}

Reply:
If the JWT is invalid, we should `throw` an error. If there's no org id in the token, return null.

Reply:
Discussed this in a Webex Teams 1:1. Seems it was decided to leave this as is based on the parent method that calls these.

Reply:
Making decisions over 1:1 chats and giving that as an answer doesn't feel satisfactory, could you explain the reasoning for making such decision?
How a works function shouldn't be dictated by how something else wants to use it, what is right for the function should determined how it is used.

I don't think we should be returning `undefined` when is a purposely set value but that's my opinion

Reply:
> Needs to be null/undefined [I prefer undefined] in this case, later we will do a:
> 
> ```js
> combinedFunctionForGettingOrgId(token = '') {
>   const foundOrgId = extractFromJWT(token) || extractFromUserToken(token);
> 
>   if (!foundOrgId) {
>     throw new Error('message');
>   }
>   ...
> }
> ```

@lalli-flores, @InteractiveTimmy went over his usage as described in the sample code above and I was convinced.

Reply:
@adamweeks thanks for the insight, I would appreciate if conversations on teams PRs happen on a team space instead of 1:1 spaces. If we want to move away from relying on one person to review everything, we should probably open discussions for the team to have an opportunity to engage